pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -165,6 +168,25 @@ func (s *statesInformer) reportNodeTopology() {
 		Policy:  cpuPolicy,
 		Options: cpuManagerOpt,
 	}
+
```

Comment:
Only one of GuessCPUManagerOptFromKubelet and GuessConfigFilePathFromKubelet will be succeed, but we should not log error in the failed path.